apps/css-workshop/pages/textarea.html

```diff
@@ -47,27 +47,31 @@ <h2>Default</h2>
         id="demo-default-singular"
       >
         <div class="iui-input-label iui-required">Textarea label</div>
-        <textarea
-          placeholder="Placeholder…"
-          class="iui-input"
-          rows="3"
-          id="test-default-singular"
-          required
-        ></textarea>
+        <div class="iui-input-with-icon">
```

Comment:
Why are we adding this wrapper to the cases where we don't have a status icon?

Reply:
Since #1753, we always wrap the `LabeledInput`'s/`LabeledTextarea`'s input/textarea in an `.iui-input-with-icon`. E.g. [`LabeledTextarea → Basic` story](https://itwin.github.io/iTwinUI/react/?story=labeledtextarea--basic&theme=dark)

<details>
<summary>Screenshot</summary>

![image](https://github.com/iTwin/iTwinUI/assets/45748283/d9f5cd08-9533-43ad-aefc-3ed601064e32)

</details>

I think this was possible because we modified the styles to handle the case where the `.iui-input-with-icon` had no icon ([example](https://github.com/iTwin/iTwinUI/pull/1753/files?file-filters%5B%5D=.scss&show-viewed-files=true#diff-4623585aa5d420e8e06bbb618dd09a22a22d7946948cd311def55eaa7a64e9a0L111-R121)).

But yes, since the class name include "with icon", I guess having an icon always is better. So, I removed the `.iui-input-with-icon` from the textarea demos when there was no end icon.

I added a TODO in the PR description to try updating the react code for `LabeledInput`/`LabeledTextarea` too.

Reply:
I'm also ok with leaving `LabeledInput`/`LabeledTextarea` unchanged for a while. There are more important things to tackle in the meantime.